postgres_test.go

```diff
@@ -8,15 +8,14 @@ import (
 
 	"github.com/db-journey/migrate/direction"
 	"github.com/db-journey/migrate/file"
-	pipep "github.com/db-journey/migrate/pipe"
 )
 
 // TestMigrate runs some additional tests on Migrate().
 // Basic testing is already done in migrate_test.go
 func TestMigrate(t *testing.T) {
-	host := os.Getenv("POSTGRES_PORT_5432_TCP_ADDR")
-	port := os.Getenv("POSTGRES_PORT_5432_TCP_PORT")
-	driverURL := "postgres://postgres@" + host + ":" + port + "/template1?sslmode=disable"
+	host := getenvDefault("POSTGRES_PORT_5432_TCP_ADDR", "localhost")
+	port := getenvDefault("POSTGRES_PORT_5432_TCP_PORT", "5432")
+	driverURL := "postgres://postgres:p@" + host + ":" + port + "/template1?sslmode=disable"
```

Comment:
there's no password when using docker, why this change?

Reply:
yeah, that should've been removed.